gui/services/models.py

```diff
@@ -318,8 +318,7 @@ class AFP(Model):
         ),
         max_length=255,
         blank=True,
-        choices=list(choices.IPChoices()),
-        default='',
+        null=True,
```

Comment:
Since,I wanted the bindip field in AFP to be similar to that of CIFS, I implemented it same as CIFS, 
But yeah, "default=' ' " would work same as "null=True".
I chose "null=True", since this is how it is implemented for CIFS bindip.